### ISSUE TITLE
d3fc-label-layout

### ISSUE THREAD

**Issue by Serg65s** [d3fc/d3fc#1661]
Is there a drag and drop option for d3fc-label-layout?

**Comment by ColinEberhardt** [comment]
No, there is not - and we don't have immediate plans to implement that I'm afraid.
